resources/views/buyers/checkout_confirm.blade.php

```diff
@@ -0,0 +1,185 @@
+@extends('layouts.app')
+
+@section('title', 'Buyer: Checkout Confirm Page')
+<style>
+    label:not(.form-check-label) {
+        margin-top: 0.5rem;
+        margin-bottom: 0rem;
+    }
+
+    .card-body img {
+        width: 50px;
+        height: 50px;
+    }
+
+    .card {
+        width: 40%;
+    }
+
+    .card-footer,
+    .card-header {
+        background-color: #f8fafc !important;
+    }
+</style>
```

Comment:
Can you put this in a separate css file?

Reply:
I put this in checkout.css with commit [643e1d0](https://github.com/kredo-school/16th_rankshty/pull/44/commits/643e1d010867e19342332621f7a2170407eea5ce).